src/main/java/console/app/dev/service/OutputFormatService.java

```diff
@@ -0,0 +1,36 @@
+package console.app.dev.service;
+
+public class OutputFormatService {
+    private int recordMaxLength;
+
+    public OutputFormatService(int recordMaxLength) {
+        this.recordMaxLength = recordMaxLength;
+    }
+
+    public void printHeader() {
+        String format = "|%1$-5s|%2$-5s|%3$-" + recordMaxLength + "s|\n";
```

Comment:
It's better to declare these symbols as constant. It will make code more readable and you will be able to reuse it